docs/filesystem_layer/stratum1.md

```diff
@@ -1,44 +1,68 @@
+
+!!! warn Potential issues with intrusion prevention systems
+    In the past we have seen a few occurrences of data transfer issues when files were being pulled in by or from a Stratum 1 server.
+    In such cases the `cvmfs_server snapshot` command, used for synchronizing the Stratum 1, may break with errors like `failed to download <URL to file>`.
+    Trying to manually download the mentioned file with `curl` will also not work, and result in errors like:
+    ```
+    curl: (56) Recv failure: Connection reset by peer
+    ```
+    In all cases this was due to an intrusion prevention system scanning the associated network, and hence scanning all files going in or out of the Stratum 1.
+    Though it was a false-positive in all cases, this breaks the synchronization procedure of your Stratum 1.
+    If this is the case, you can try switching to HTTPS by using `https://aws-eu-west-s1-sync.eessi.science` for synchronizing your Stratum 1.
+    Even though there is no advantage for CVMFS itself in using HTTPS (it has built-in mechasnims for ensuring the integrity of the data),
+    this will prevent the described issues, as the intrusion prevention system will not be able to inspect the encrypted data.
 
-Finally, make sure that ports 80 (for the Apache web server) and 8000 are open.
+### Manual configuration
 
+In order to set up a Stratum 1 manually, you can make use of the instructions in the [Private Stratum 1 replica server](https://multixscale.github.io/cvmfs-tutorial-hpc-best-practices/access/stratum1/)
+section of the MultiXscale tutorial ["Best Practices for CernVM-FS in HPC"](https://multixscale.github.io/cvmfs-tutorial-hpc-best-practices/).
 
-## Step 1: set up the Stratum 1
+### Configuration using Ansible
 
 The recommended way for setting up an EESSI Stratum 1 is by running the Ansible playbook `stratum1.yml`
 from the [filesystem-layer repository on GitHub](https://github.com/EESSI/filesystem-layer).
+For the commands in this section, we are assuming that you cloned this repository, and your working directory is `filesystem-layer`.
 
-Installing a Stratum 1 requires a GEO API license key, which will be used to find the (geographically) closest Stratum 1 server for your client and proxies.
-More information on how to (freely) obtain this key is available in the CVMFS documentation: https://cvmfs.readthedocs.io/en/stable/cpt-replica.html#geo-api-setup.
+!!! note GEO API
+    Installing a Stratum 1 usually requires a GEO API license key, which will be used to find the (geographically) closest Stratum 1 server for your client and proxies.
+    However, for a private Stratum 1 this can be skipped, as clients should just connect to your local Stratum 1 by default.
+    
+    If you do want to set up the GEO API, you can find more information on how to (freely) obtain this key in the CVMFS documentation: https://cvmfs.readthedocs.io/en/stable/cpt-replica.html#geo-api-setup.
```

Comment:
Thing is: the GEO api might not be able to tell apart e.g. if a Stratum 1 in groningen or stratum 1 in Amsterdam is closer to a client in Amsterdam, but the latency would probably show the difference. The advantage over hardcoding (as you do below) is that if the private stratum 1 is down, the fallback option is _the second closest_ (in a latency sense), which is probably the best fallback.

Reply:
I think it can only do it based on distance/geography. And since it does introduce some overhead (hopefully your local server is the geographically closest one :smile:), even the CVMFS devs recommended to switch it off in such cases.

Reply:
Clients should automatically connect to the local one anyway if it's the first one in the list, and if that one goes down, they will still use the GEO API of one of the public Stratum 1 servers to find the closest one.

Reply:
There have been only recent tentative ideas about implementing a latency-based mechanism, nothing concrete. Also the GeoDB can be rather fallible and inaccurate.

CVMFS_USE_GEOAPI can be enabled on a repository basis, but not by server AFAIK. So a client should do either order-based OR geo-based selection. But I think (not sure) there should be logic somewhere that a RFC1918 private network address should always be considered closest, if you are using a private IP on a local s1.

Reply:
Reworded this part a bit in https://github.com/EESSI/docs/pull/157/commits/40668b1d66756c9513b193edff95b1b4fca88cfc.